export/export_pipeline/compute_collision_representation.py

```diff
@@ -0,0 +1,427 @@
+"""
+This script decomposes all static meshes into sets of convex regions.
+Decomposition method and parameters are based on the parameters chosen in UE, which are captured in 
+gltf_scene/actors_information.json.
+"""
+import argparse
+from collections import deque, defaultdict
+import copy
+import json
+import multiprocessing as mp
+if mp.current_process().name == 'MainProcess':
+  import bpy
+import coacd  # needs to be imported after bpy, otherwise terminate called after throwing an instance of 'std::bad_cast'
+import numpy as np
+import os
+import shutil
+import subprocess
+import trimesh
+import utils
+import yaml
+
+
+osp = os.path
+
+
+def decimation_lookup(poly_count):
```

Comment:
- Comment is more clear than the function name, rename to `get_adaptive_decimation_ratio(poly_count)` and delete comment.
- Implement this function using two arrays and the `bisect` function so the code is smaller and cleaner.